hypothesis/h
# Remove NGINX from the separate websocket only

For:

 * https://github.com/hypothesis/h/issues/7090

## What this PR does

 * NGINX has been removed from the separate websocket image
 * NGINX is still running in the main image though
 * This means we need two sets of config for the different production images
 * Also fixed a small bug where the `make run-docker-ws` ran on the wrong port to allow easier testing

## Testing notes

The usual test for the WS is to open two pages on the same site and annotate and see that the notification shows up. You will need to run the `client`, and probably `via` + `viahtml` for this.

Check the main image:

 * `make docker`
 * `make run-docker`
 * Check the WS

Check the WS separate image:

 * Shut it down the main image if running
 * `make docker-ws`
 * `make run-docker-ws`
 * `tox -e dev --run-command "pserve --reload conf/development-app.ini"` - Run the main web app
 * Check the WS
